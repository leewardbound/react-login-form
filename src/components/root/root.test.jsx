import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Root from '.';

const renderer = new ShallowRenderer();

describe('Root', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <Root />
    );

    expect(component).toMatchSnapshot();
  });
});
