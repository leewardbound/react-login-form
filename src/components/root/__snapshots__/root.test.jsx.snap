// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root should render correctly 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(Symbol.observable): [Function],
    }
  }
>
  <ThemeProvider
    theme={
      Object {
        "bpLarge": "1200px",
        "bpMedium": "992px",
        "bpSmall": "768px",
        "cols": 12,
        "errorColor": "tomato",
        "gutter": "15px",
        "headingColor": "lightgrey",
        "primaryColor": "lightgrey",
        "successColor": "green",
        "textColor": "#666",
        "wLarge": "1170px",
        "wMedium": "970px",
        "wSmall": "750px",
      }
    }
  >
    <LoginScreen />
  </ThemeProvider>
</Provider>
`;
